tests/suites/test_suite_entropy.function

```diff
@@ -381,5 +381,8 @@ void entropy_nv_seed( char *read_seed_str )
 void entropy_selftest( )
 {
     TEST_ASSERT( mbedtls_entropy_self_test( 0 ) == 0 );
```

Comment:
This should be a test called within `mbedtls_entropy_self_test()`, not it's own self-test - although it should be available through the API (e.g. not static) if a user wants to call it.

Therefore please remove from this module and from selftest.c.